CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(CMAKE_CXX_STANDARD 20)
 # find dependencies
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(backward_ros REQUIRED)
```

Comment:
Do we really want to make this required? For all other dependencies we add them via submodules / FetchContent, whereas backward_ros would be expected to be already part of the used workspace. 

Since rig_reconfigure is fully usable without it, wouldn't be "quiet" enough? In case the package is available we'll link it in but we do not expect all users to have it in their workspace. Not sure how we would reflect this within the package.xml